app/(main)/league/[leagueId]/entry/all/page.test.tsx

```diff
@@ -149,7 +150,6 @@ describe('League entries page (Entry Component)', () => {
 
     expect(entryPageHeader).toBeInTheDocument();
     expect(entryPageHeaderToLeaguesLink).toBeInTheDocument();
-    expect(entryPageHeaderToLeaguesLink).toHaveAttribute('href', '/league/all');
```

Comment:
Hey Ryan, just curious, what was the reason for removing this?

Reply:
I removed this as I moved the `data-testid` to the containing `div` rather than the `CustomLink` component to be aligned with the implementation Shashi has asked for previously. As such, the `div` does not have an `href` attribute 😅

Reply:
Exactly :)